CHANGELOG.md

```diff
@@ -30,11 +35,15 @@ Changed
 - Store git sha in salt/_version.py when installing from a tag so it can be found if needed later. (#59137)
 - Changed package manager detection in yumpkg module (#59201)
 - Updating the pkg beacon to fire the events when there are upgrades to packages, but also when watched packages are installed or removed. Breaking out the logic for listing pkgs from context into a separate function to aid in testing. Updating tests to ensure context is not used when use_context option to list_pkgs is False. (#59463)
+- Store git sha in salt/_version.py when installing from a tag so it can be found if needed later. (#59137)
 
 
 Fixed
 -----
 
+- When instantiating the loader grab values of grains and pillars if
```

Comment:
These are not included in the 3002.6 release. I think the 3002.6 header mistakingly got added in the wrong  place. It should reflect the same content in doc/topics/releases/3002.6.rst

Reply:
Fixed